import.qmd

```diff
@@ -1,41 +1,41 @@
-# Import {#sec-import .unnumbered}
+# Importar {#sec-import .unnumbered}
 
 ```{r}
 #| echo: false
 
 source("_common.R")
 ```
 
-In this part of the book, you'll learn how to import a wider range of data into R, as well as how to get it into a form useful for analysis.
-Sometimes this is just a matter of calling a function from the appropriate data import package.
-But in more complex cases it might require both tidying and transformation in order to get to the tidy rectangle that you'd prefer to work with.
+Nesta parte do livro, você aprenderá como importar uma vasta variedade de dados para o R, assim como deixá-los em uma forma útil para suas análises.
+Algumas vezes, isto é apenas uma questão de chamar uma função de um pacote de importação de dados apropriado.
+Mas em casos mais complexos, será necessário tanto a organização e transformação dos dados para se obter o formato retangular (*tidy*) que você gostaria de trabalhar.
 
 ```{r}
 #| label: fig-ds-import
 #| echo: false
 #| fig-cap: |
-#|   Data import is the beginning of the data science process; without
-#|   data you can't do data science!
+#|   Importação dos dados é o início do nosso modelo de processo da ciência de dados; sem
+#|   dados você não pode praticar ciência de dados!
 #| fig-alt: |
-#|   Our data science model with import highlighted in blue.
+#|   Nosso modelo de processo de ciência de dados com Importar destacado em azul.
 #| out.width: NULL
 
 knitr::include_graphics("diagrams/data-science/import.png", dpi = 270)
 ```
 
-In this part of the book you'll learn how to access data stored in the following ways:
+Nesta parte do livro você aprenderá como acessar dados armazenados das seguintes maneiras:
 
--   In @sec-import-spreadsheets, you'll learn how to import data from Excel spreadsheets and Google Sheets.
+-   No @sec-import-spreadsheets, você aprenderá como importar dados de planilhas do Excel e do Planilhas Google (*Google Sheets*).
 
--   In @sec-import-databases, you'll learn about getting data out of a database and into R (and you'll also learn a little about how to get data out of R and into a database).
+-   No @sec-import-databases, você aprenderá sobre obter dados de um banco de dados (*databases*) para o R (e você também aprenderá um pouco sobre como exportar dados do R para um banco de dados).
```

Comment:
Na linha trinta em inglês diz: "about getting data out of a dataset *and* into R". Gostaria de confirmar se a tradução não seria "sobre obter dados de um banco de dados e dentro do R". Para mim o texto não fica claro. "Aprender sobre retirar um dado de um banco de dados e inserir dentro do R." Faz sentido?

Reply:
Oieee! Fiz uma modificação para ficar mais claro esse fluxo :D